src/domain/journey/space/createSpace/plansTable/usePlanAvailability.ts

```diff
@@ -0,0 +1,30 @@
+import { useAccountsHostsQuery } from '../../../../../core/apollo/generated/apollo-hooks';
+import { useUserContext } from '../../../../community/user';
+
+interface Provided {
+  loading: boolean;
+  isFreePlanAvailable: () => boolean;
+}
+
+export const usePlanAvailability = ({ skip }: { skip?: boolean }): Provided => {
+  const { user: currentUser, loading: loadingUser } = useUserContext();
+
+  const { data, loading: loadingHosts } = useAccountsHostsQuery({
+    skip,
+  });
+
+  const isFreePlanAvailable = () => {
+    if (loadingUser || loadingHosts) {
+      return false;
+    }
+    if (!currentUser?.user.id) {
+      return false;
+    }
+    const userIsHost = data?.accounts.some(account => account.hosts?.some(host => host.id === currentUser.user.id));
```

Comment:
Please leave a comment here that this is temporary and the logic will block the availability of a FREE one if the user has a trial or paid one.